lib/openai/runs.rb

```diff
@@ -4,8 +4,8 @@ def initialize(client:)
       @client = client.beta(assistants: "v1")
     end
 
-    def list(thread_id:)
-      @client.get(path: "/threads/#{thread_id}/runs")
+    def list(thread_id:, parameters: {})
```

Comment:
Ideally I'd make matching changes to `runs_spec.rb` -- are there ways that you generate the cassette? (I'm looking to avoid doing this manually)